frontend/src/pages/EventSignInPage/index.tsx

```diff
@@ -9,6 +9,7 @@ import HKN_TRIDENT_LOGO from '@Images/hkn-trident.png';
 import { Loading, Card, PublicPageLayout } from '@SharedComponents';
 import { getEventById, signInToEvent } from '@Services/EventService';
 import { EventResponse, AppUserEventRequest } from '@Services/api/models';
+import { format, parseISO } from 'date-fns';
```

Comment:
Please name your branch after the task you are working on i.e. `add_static_event_info` and not something like what you have. Don't use your own name and it's best to not add "branch" into the branch name as well and use the space for other words more descriptive of your task